generators/ruby/faraday/templates/README.mustache

```diff
@@ -1,40 +1,42 @@
-# Onfido
+# Onfido Ruby Library
 
 The official Ruby library for integrating with the Onfido API.
 
-[![Gem Version](https://badge.fury.io/rb/onfido.svg)](http://badge.fury.io/rb/onfido)
-[![Build Status](https://travis-ci.org/onfido/onfido-ruby.svg?branch=master)](https://travis-ci.org/onfido/onfido-ruby)
+Documentation can be found at <{{ documentationUrl }}>.
 
-Documentation can be found at https://documentation.onfido.com
+This version uses Onfido API {{ apiVersion }}. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
 
-This version uses Onfido API v3.6 and is compatible with Ruby 2.4 onwards. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
+[![Gem Version](https://badge.fury.io/rb/onfido.svg)](http://badge.fury.io/rb/onfido)
+[![Build Status](https://travis-ci.org/onfido/onfido-ruby.svg?branch=master)](https://travis-ci.org/onfido/onfido-ruby)
 
-## Installation
+## Installation & Usage
 
-Add this line to your application's Gemfile:
+### Installation
 
 ```ruby
-gem 'onfido', '~> 2.0.1'
+gem {{{gemName}}}, '~> 2.0.1'
 ```
 
-## Getting started
-
-Configure with your API token and region:
+Configure with your API token, region and optional timeout (default value is 30):
 
 ```ruby
-onfido = Onfido::API.new(
-  api_key: ENV['ONFIDO_API_KEY'],
-  # Supports :eu, :us and :ca. Previously defaulted to :eu.
-  region: :eu
-)
+require {{{gemName}}}
+
+Onfido.configure do |config|
+  config.api_token = ENV["ONFIDO_API_TOKEN"]
+  config.region = config.region[:EU]
+  config.timeout = 30
+end
+
+onfido_api = Onfido::DefaultApi.new
 ```
 
 All resources share the same interface when making API calls. Use `.create` to create a resource, `.find` to find one, and `.all` to fetch all resources.
 
-For example, to create an applicant:
+### Making a call to the API
```

Comment:
To be coherent with the other libraries, let write the default timeout down:

`NB: by default, timeout values are set to 30 seconds.`